web-js/4.x/download-csv/README.md

```diff
@@ -0,0 +1,2 @@
+## Download Client Side CSV
```

Comment:
https://raw.githubusercontent.com/Esri/developer-support/master/web-js/4.x/add-attachments-with-42/README.md

a live demo link would be a great addition to the README. the one below doesn't work now, but it will when the code hits `master`

https://esri.github.io/developer-support/web-js/4.x/download-csv/downloadCSV.html

(I'd rename it to `index.html` for brevity, but that one's up to you).

Reply:
thanks for the suggestions, added both